components/PriceCard.tsx

```diff
@@ -1,11 +1,13 @@
+import { useState, useEffect } from 'react';
 import { CheckIcon } from '@heroicons/react/outline';
 import { ReactNode } from 'react';
 import { Button } from './Button';
 
 interface Props {
     name: string;
+    noDescription?: boolean;
```

Comment:
what this for

Reply:
by default it calculates the pricing in this component because the price is being fetched here and shows the $18.50 per week thing. This says don't show it

Reply:
why is it description then if its to do with price

Reply:
because you can pass in a description here too